library/src/main/java/com/chuckerteam/chucker/internal/support/ContextExt.kt

```diff
@@ -6,17 +6,17 @@ import com.google.android.material.dialog.MaterialAlertDialogBuilder
 
 internal fun Context.showDialog(
     dialogData: DialogData,
-    onPositiveClick: (() -> Unit)?,
-    onNegativeClick: (() -> Unit)?
+    onPositiveClick: () -> Unit = {},
+    onNegativeClick: () -> Unit = {}
```

Comment:
I'm ok with default arguments but why make them not nullable?

Reply:
If the arguments are nullable and with default value, look like `(() -> Unit)? = null`, it's chatty and not neat.

Reply:
What's chatty and not neat and by which metric? Having nullable arguments saves object allocations.

Reply:
Ok, for needing not to allocate objects, I will change them to nullable.